README.md

```diff
@@ -116,16 +116,16 @@ export interface Asset {
   tokenId: string | null,
   // The asset's contract address
   tokenAddress: string,
-  // The Wyvern schema name (defaults to "ERC721") for this asset
-  schemaName?: WyvernSchemaName,
+  // The schema name (defaults to "ERC721") for this asset
+  schemaName?: SchemaName,
```

Comment:
nit: should we call this ```TokenType```? or something similar? SchemaName doesn't seem like the best to me

Reply:
Renamed to TokenStandard